assets/Js/word-count-calculator.js

```diff
@@ -16,11 +16,25 @@ function count_alphabets(val){
   return 0;
 }
 
+function longest(val){
+       
+  let sentence=val.split(" ");
+    let longestWord=sentence.reduce(function(a,b){
+      if(a.length>b.length) return a;
+      return b;
+    },"");
+    return longestWord;
+}
+// var temp=document.getElementById("textcontent");
+temp=document.getElementById("textcontent");
```

Comment:
The variable `temp` is declared but not used in the provided code. Therefore, it can be safely removed without affecting the functionality of the code.

Or you can tell me the purpose of this variable.